test/test_masks_to_boxes.py

```diff
@@ -0,0 +1,34 @@
+import os.path
+
+import PIL.Image
+import numpy
+import torch
+
+import torchvision.ops
+import torchvision.ops.boxes
```

Comment:
I think you only need 1 import.
```suggestion
from torchvision.ops import masks_to_boxes.
```